toolkit/themes/shared/design-system/build/css/tokens-brand.css

```diff
@@ -0,0 +1,5 @@
+:root {
```

Comment:
Do we not need `@import url("chrome://global/skin/design-system/tokens-shared.css");` here anymore?